sql/group.go

```diff
@@ -72,6 +72,7 @@ func (p *planner) groupBy(n *parser.Select, s *scanNode) (*groupNode, error) {
 	}
 
 	group := &groupNode{
+		planner: p,
```

Comment:
We seem to have eventually required the whole planner in other nodes. Do you see specific harm in giving the node the whole planner?